docs/transaction-api.asciidoc

```diff
@@ -69,6 +69,14 @@ Defaults to `unnamed`
 You can alternatively set this via <<span-type,`span.type`>>.
 Defaults to `custom`
 
+* `options` - The following options are supported:
+
+** `startTime` - The time when the span started.
```

Comment:
I didn't document the ability to set `traceparent` here as we hadn't done that previously. LMKWYT